sh.fhs.KatawaShoujoReEngineered.yaml

```diff
@@ -0,0 +1,110 @@
+app-id: sh.fhs.KatawaShoujoReEngineered
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: "ksre"
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    add-ld-path: "."
+    version: "22.08"
+    autodelete: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does it support wayland? If yes then fallback-x11 on line 14. otherwise remove this.

Reply:
I tried it on wayland, got segfault, removed wayland socket